src/platform/Linux/ConfigurationManagerImpl.cpp

```diff
@@ -365,5 +365,25 @@ CHIP_ERROR ConfigurationManagerImpl::GetLocationCapability(uint8_t & location)
     return err;
 }
 
+CHIP_ERROR ConfigurationManagerImpl::GetCountryCode(char * buf, size_t bufSize, size_t & codeLen)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    VerifyOrReturnError(bufSize > 2, CHIP_ERROR_BUFFER_TOO_SMALL);
+
+    if (PosixConfig::ConfigValueExists(PosixConfig::kConfigKey_CountryCode))
+    {
+        err = PosixConfig::ReadConfigValueStr(PosixConfig::kConfigKey_CountryCode, buf, bufSize, codeLen);
+    }
+    else
+    {
+        strncpy(buf, "XX", bufSize);
```

Comment:
Is this defined by the spec? Otherwise, maybe this should be guarded with `#if CHIP_CONFIG_TEST` like in other places where test values are used?

Reply:
It is, CHIP-Specifications/connectedhomeip-spec#4758